web/app.js

```diff
@@ -197,10 +204,11 @@ var PDFViewerApplication = {
     this.pdfViewer = new PDFViewer({
       container: container,
       viewer: viewer,
+      eventBus: eventBus,
       renderingQueue: pdfRenderingQueue,
       linkService: pdfLinkService,
       downloadManager: downloadManager
-    });
+    }, eventBus);
```

Comment:
`PDFViewer` takes only one parameter `options` and you're already passing the event bus in the options object, so this change does not appear to be required.